g2p/mappings/langs/eng/config.yaml

```diff
@@ -12,4 +34,16 @@ mappings:
     mapping: eng_ipa_to_arpabet.json
     rule_ordering: apply-longest-first
     <<: *shared
+  - display_name: English IPA to Arpabet
```

Comment:
Several sets of mappings in generated also end up with the same name. Maybe we don't care? I guess nothing actually depends on these display names, but I'm still worried it could get confusing.

Reply:
Never mind, it's the same mapping for both, so it's fair to use the same name.